src/aosm/azext_aosm/inputs/vhd_file_input.py

```diff
@@ -82,3 +83,14 @@ def get_schema(self) -> Dict[str, Any]:
 
         logger.debug("Schema for VHD file input: %s", json.dumps(schema, indent=4))
         return copy.deepcopy(schema)
+
+    def _format_default_config(self):
+        print(self.default_config)
+        formatted_config = {}
+        for (key, value) in self.default_config.items():
+            # This must be an integer, but is a string in the input file
+            if key == "image_disk_size_GB":
+                value = int(value)
+            formatted_key = snake_case_to_camel_case(key)
+            formatted_config[formatted_key] = value
+        return formatted_config
```

Comment:
Is it not easier to do all this in the __init__(), perhaps like this:

```
         if "image_disk_size_GB" in self.default_config:
             self.default_config["imageDiskSizeGB"] = int(self.default_config["image_disk_size_GB"])
             del self.default_config["image_disk_size_GB"]
```

(and if we know we always have `image_disk_size_GB`, then we don't need the `if` statement)
(and if it doesn't matter if the old value sticks around, don't need the `del` either)

Reply:
we dont always have disk size + i like having a separate dict returned so we dont have to worry about deleting things